index.js

```diff
@@ -1,6 +1,21 @@
+const fs = require('fs');
 const Discord = require('discord.js');
-const client = new Discord.Client();
+const Keyv = require('keyv');
 const settings = require('./config.json');
+const functions = require('./functions');
+
+const client = new Discord.Client({ partials: ['MESSAGE', 'REACTION'] });
+client.commands = new Discord.Collection();
```

Comment:
I'm definitely a typed type of guy, and mutating an existing type to include
custom information would be quite interesting to me. I don't think I'd do it.
I would probably prefer to keep the data seperate, so that people looking at
other files and finding this strange "client" object to have very custom properties
do not get surprised.

Reply:
Yeah, just easier to pass that stuff through files when it's all part of the client object. If it get's particularly complicated, I'd document this somewhere.